### DISCUSSION TITLE
Viewing data logged via _log.debug

### DISCUSSION THREAD

**Discussion by bfailing** [artisan-roaster-scope/artisan#972] [Q&A] [answered]
How do you view data logged via the `_log.debug` commands found in the python files?

**Comment by MAKOMO** [comment, marked as answer]
The location of the ```artisan.log``` file depends on the platform Artisan is running on.

* macOS : ```~/Library/Application Support/artisan-scope/Artisan/```
* Windows: ```C:\Users\<USER>\AppData\Local\artisan-scope\Artisan\```
* Linux: ```~/.local/shared/artisan-scope/Artisan/```

An OPTION click to the plus icon opens an email message with the log files attached and an OPTION-COMMAND click toggles the debug mode as described under menu ```Help >> Keyboard Shortcuts```

**Reply by bfailing** [reply]
The `Help > Keyboard Shortcuts` dialog doesn't show the "Addl Shortcuts" (which includes the shortcut for toggling the debug mode) and "Menu Shortcuts" in this XSLX file here: https://github.com/artisan-roaster-scope/artisan/blob/master/doc/help_dialogs/Input_files/keyboardshortcuts.xlsx.  How are those accessed in the Artisan program?

**Reply by MAKOMO** [reply]
The additional shortcuts are rendered below the shortcuts in the next version of Artisan (and the current development builds).
